test/libsolidity/syntaxTests/nameAndTypeResolution/219_memory_arrays_not_resizeable.sol

```diff
@@ -5,4 +5,4 @@ contract C {
     }
 }
 // ----
-// TypeError: (72-80): Memory arrays cannot be resized.
+// TypeError: (72-80): Member "length" is read-only and cannot be used to resize arrays.
```

Comment:
Is there a test for ``x.push()`` to fail for memory arrays?

Reply:
Now there is :-)